src/Microsoft.DotNet.Build.Tasks/PackageFiles/optional-tool-runtime/optional.json

```diff
@@ -2,6 +2,7 @@
   "frameworks": {
     "net45": {
       "dependencies": {
+        "Microsoft.SymbolUploader.Build.Task": "1.0.0-alpha-00001",
```

Comment:
Where can I get the packages this package depends on? I can't find them on the VSTS feed or on any public feeds.

Reply:
Are those dependencies going to cause a problem?  I'm using the msbuild pack logic which doesn't seem to have a simple way to suppress them unless I want to manually create the nuspec file.

Reply:
I'd expect the tool restore to fail when it can't find them. I'm not aware of a small way to get rid of them either (but it's not an area I know much about).